src/Maestro/maestro-angular/src/app/page/build/build.component.ts

```diff
@@ -311,15 +334,31 @@ export class BuildComponent implements OnInit, OnChanges {
       );
   }
 
-  public getRepo(build: Build) {
-    return build.gitHubRepository || build.azureDevOpsRepository;
+  public getOngoingBuildInfo(build: Build): Observable<string | null> {
```

Comment:
It's not a huge issue but it would probably be better to name it something like `getOngoingBuildUrl` because this is what this method does from what I see.